src/wp-includes/js/autosave.js

```diff
@@ -449,7 +449,7 @@ window.autosave = function() {
 			 */
 			function compare( str1, str2 ) {
 				function removeSpaces( string ) {
-					return string.toString().replace(/[\x20\t\r\n\f]+/g, '');
+					return string.toString().replace(/[\x20\t\r\n\f]+/g, '').replace(/\s+/g, '').replace(/&nbsp;+/g, '');
```

Comment:
I think this line can be simplified a little to:
`return string.toString().replace(/[\x20\t\r\n\f\s]+/g, '').replace(/&nbsp;+/g, '');`

So, combine the `\s` special group into the first regex expression. I haven't found a way so far to incorporate `&nbsp;` also.